MediaBrowser.Providers/Manager/MetadataService.cs

```diff
@@ -164,7 +164,7 @@ public virtual async Task<ItemUpdateType> RefreshMetadata(BaseItem item, Metadat
             }
 
             // Next run remote image providers, but only if local image providers didn't throw an exception
-            if (!localImagesFailed && refreshOptions.ImageRefreshMode != MetadataRefreshMode.ValidationOnly)
+            if (!localImagesFailed && refreshOptions.ImageRefreshMode > MetadataRefreshMode.ValidationOnly)
```

Comment:
I'm not a fan of using `>` when comparing enums since it can have unintended consequences when adding a new value

Reply:
I do agree but we currently use that kind of comparison for this enum quite often, so it's fine I think.